stacks-node/src/burnchains/rpc/rpc_transport/mod.rs

```diff
@@ -176,8 +176,10 @@ impl RpcTransport {
     /// # Arguments
     ///
     /// * `id` - A unique identifier for correlating responses.
-    /// * `method` - The name of the JSON-RPC method to invoke.
+    /// * `relative_path` - An optional relative path to append to the transport base path for this request.
+    ///                     If `None`, the base path is used. Leading `/` in the path is handled automatically.
     /// * `params` - A list of parameters to pass to the method.
+    /// * `method` - The name of the JSON-RPC method to invoke.
```

Comment:
Nit: `method` should go before `params` in this list, just to match the list of args in the function.